### DISCUSSION TITLE
About demos

### DISCUSSION THREAD

**Discussion by Kerolos838** [nolimits4web/swiper#6784] [General]
The demos don't work on the site or with me
Do this prob will fix in the next version or when?
![EX: photo from the site](https://github.com/nolimits4web/swiper/assets/88841322/c37920b3-aad8-44ed-b8a9-1c8d2147cb0e)

**Comment by Arhell** [comment]
working, using chrome latest

**Reply by Kerolos838** [reply]
Can you take a picture for demo in your browser Because I using Chrome and edge on my phone and laptop, I have the same prob

**Reply by Arhell** [reply]
![Screenshot_1](https://github.com/nolimits4web/swiper/assets/26163841/a9273dd7-ac11-4596-b736-9b0ee71a12bc)
![Screenshot_2](https://github.com/nolimits4web/swiper/assets/26163841/73c3000c-1d81-4cae-b25e-40f8f1c6f569)
